src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -389,52 +397,161 @@ sub-<label>/[ses-<label>/]
-corresponding `dwi.nii` file, and not the magnet's coordinate system.
-Thus, any rotations applied to `dwi.nii` also need to be applied to the
-corresponding bvec file.
-
-bvec example:
+distinguish different sets of parameters.
+
+The OPTIONAL [`dir-<label>`](../99-appendices/09-entities.md#dir)
+key/value pair corresponds to a custom label the user may use to
+distinguish different sets of phase-encoding directions.
+
+**Combining multi- and single-band acquisitions**.
+The single-band reference image MAY be stored with suffix `sbref` (for example,
+`dwi/sub-control01_sbref.nii[.gz]`) as long as the image has no corresponding
+[gradient information (`[*_]dwi.bval` and `[*_]dwi.bvec` sidecar files)](#required-gradient-orientation-information)
+to be stored.
+
+Otherwise, if some gradient information is associated to the single-band diffusion
+image and a multi-band diffusion image also exists, the `acq-<label>` key/value pair
+MUST be used to distinguish both images.
+In such a case, two files could have the following names:
+`sub-01_acq-singleband_dwi.nii.gz` and `sub-01_acq-multiband_dwi.nii.gz`.
+The user is free to choose any other label than `singleband` and
+`multiband`, as long as they are consistent across subjects and sessions.
+
+### REQUIRED gradient orientation information
+
+The REQUIRED gradient orientation information corresponding to a DWI acquisition
+MUST be stored using `[*_]dwi.bval` and `[*_]dwi.bvec` pairs of files.
+The `[*_]dwi.bval` and `[*_]dwi.bvec` files MAY be saved on any level of the directory structure
+and thus define those values for all sessions and/or subjects in one place (see
+[the inheritance principle](../02-common-principles.md#the-inheritance-principle)).
+
+As an exception to the [common principles](../02-common-principles.md#definitions)
+that parameters are constant across runs, the gradient table information (stored
+within the `[*_]dwi.bval` and `[*_]dwi.bvec` files) MAY change across DWI runs.
+
+**Gradient orientation file formats**.
+The `[*_]dwi.bval` and `[*_]dwi.bvec` files MUST follow the
+[FSL format](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide#DTIFIT):
+The `[*_]dwi.bvec` file contains 3 rows with *N* space-delimited floating-point numbers
+(corresponding to the *N* volumes in the corresponding NIfTI file.)
+The first row contains the *x* elements, the second row contains the *y* elements and
+the third row contains the *z* elements of a unit vector in the direction of the applied
+diffusion gradient, where the *i*-th elements in each row correspond together to
+the *i*-th volume, with `[0,0,0]` for *non-diffusion-weighted* (also called *b*=0 or *low-b*)
+volumes.
+Following the FSL format for the `[*_]dwi.bvec` specification, the coordinate system of
+the *b* vectors MUST be defined with respect to the participant (in other words, defined
+by the axes of the corresponding `_dwi` NIfTI file) and not the magnet's coordinate system.
```

Comment:
```suggestion
the *b* vectors MUST be defined with respect to the head coordinantes in the corresponding `_dwi` NIfTI file and not the scanner's device coordinate system (see [Coordinate systems](08-coordinate-systems.md)).
```

Reply:
And... btw, is "scanner's device coordinate system" satisfactorily clear in `../08-coordinate-systems.md`?  All that is said is
>  all data acquisition starts with "device coordinates" (scanner), which does not have to be identical to the initial "file format coordinates" (DICOM), which are again different from the "head" coordinates (for example, NIFTI).

Reply:
Thanks for this suggestion. I've taken the liberty of editing your suggestion to fix a little typo ("coordinante") and introduce new lines (I had also fixed the link in parallel to your edit).

Regarding this: as I commented above, I think we should not make very deep changes regarding the gradients, as I think it will more clear for everyone to tackle them in full within #352.